README.md

```diff
@@ -1 +1,39 @@
-# Portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Portfolio
+
+## Built With
```

Comment:
- [ ] [Optional] Kindly consider adding a description for the Project in your README so your documentation looks more professional